.fleek.json

```diff
@@ -0,0 +1,7 @@
+{
+  "build": {
+    "image": "fleek/create-react-app:node-16",
```

Comment:
FYI, the README says that node 14 is required. I am unsure what version is used for the production build, but we should mirror that.